logstash-core/benchmarks/build.gradle

```diff
@@ -66,13 +64,19 @@ task jmh(type: JavaExec, dependsOn: [':logstash-core-benchmarks:clean', ':logsta
   def include = project.properties.get('include', '')
 
   doFirst {
-    args = [
+    def arg = [
             "-Djava.io.tmpdir=${buildDir.absolutePath}",
             "-XX:+UseParNewGC", "-XX:+UseConcMarkSweepGC", "-XX:CMSInitiatingOccupancyFraction=75",
```

Comment:
we can actually just remove the flag that opts into ParNewGC regardless of which JVM we are running on, since this has been a default JVM configuration since Java 8.

See: https://github.com/elastic/logstash/pull/10279/commits/9875063993e366d0a349dcaf964390a7b762810d

Reply:
Done